packages/react-cosmos-component-playground/src/components/FixtureList/index.jsx

```diff
@@ -33,28 +34,42 @@ const getFilteredFixtures = (fixtures, searchText) => {
   }, {});
 };
 
+const isFixtureSelected = (fixtures, component, fixture) => {
```

Comment:
`isExistingFixtureSelected` would be a better name because we check if a fixture is selected *and* if it exists.